client/src/components/questions/AnswerList.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useState, useEffect} from 'react';
+import Answer from './Answer.jsx';
+import api from '../../lib/api.js';
+import StyleLib from '../../assets/Stylesheet.jsx';
+
+const AnswerList = ({answers, helpfulCB}) => {
+  const answersArray = Object.keys(answers);
+  const [displayedAnswers, setDisplayedAnswers] = useState([]);
+  const [numberOfAnswers, setNumberOfAnswers] = useState(2);
+  const [buttonText, setButtonText] = useState('');
+
+  useEffect(() => {setDisplayedAnswers(answersArray.slice(0, 2))}, []);
+
+  useEffect(() => {
+    updateDisplayedAnswers();
+  }, [numberOfAnswers]);
+
```

Comment:
Nit: I noticed that you have two separate useEffects that you could possibly condense in to one. 
Instead of using a separate useEffect like you're doing here: `  useEffect(() => {
    updateDisplayedAnswers();
  }, [numberOfAnswers]);` that calls a function, just reuse the useEffect on line 12 `useEffect(() => {setDisplayedAnswers(answersArray.slice(0, 2))}, []);` and include the dependency `[numberOfAnswers]` and replace `2` with state  `numberOfAnswers` . 
  
  Then you can get rid of this function on line 26 that essentially does the same thing. 
  `  const updateDisplayedAnswers = () => {
    setDisplayedAnswers(answersArray.slice(0, numberOfAnswers));
  }`